tensorflow/core/common_runtime/sycl/sycl_allocator.cc

```diff
@@ -19,7 +19,7 @@ limitations under the License.
 
 namespace tensorflow {
 
-SYCLAllocator::SYCLAllocator(Eigen::QueueInterface *queue)
+SYCLAllocator::SYCLAllocator(Eigen::QueueInterface* queue)
```

Comment:
In the future, please keep large formating changes and functional changes in separate PRs.